tests/openvino/test_diffusion.py

```diff
@@ -104,12 +111,14 @@ def test_load_vanilla_model_which_is_not_supported(self):
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
     @require_diffusers
     def test_ov_pipeline_class_dispatch(self, model_arch: str):
-        auto_pipeline = self.AUTOMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
+        auto_cls = self.get_auto_cls(model_arch)
+        auto_pipeline = DiffusionPipeline if model_arch != "sana" else auto_cls
+        auto_pipeline = auto_cls.from_pretrained(MODEL_NAMES[model_arch])
         ov_pipeline = self.OVMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
 
         self.assertEqual(ov_pipeline.auto_model_class, auto_pipeline.__class__)
 
-        auto_pipeline = DiffusionPipeline.from_pretrained(MODEL_NAMES[model_arch])
+        auto_pipeline = auto_pipeline.from_pretrained(MODEL_NAMES[model_arch])
```

Comment:
why ? `DiffusionPipeline` should be able to return the right SanaPipeline

Reply:
unfortunatly, only on main branch https://github.com/huggingface/diffusers/pull/10538
diffusers pypi package (0.32.2) does not have sana in autopipeline registry

Reply:
hmm yeah for autopipeline I see but `DiffusionPipeline` is not part of the autopipeline registery, it's the main entrypoint of all pipelines (it just reads the `_class_name` in https://huggingface.co/Efficient-Large-Model/Sana_1600M_1024px_BF16_diffusers/blob/main/model_index.json#L2)